server.js

```diff
@@ -6,12 +6,15 @@ var app = express();
 
 const rootDir = path.join(__dirname, 'build');
 app.use(express.static(rootDir));
-app.get('/*', (req, res) => {
+app.get('/*splat', (req, res) => {
   res.sendFile('index.html', { root: rootDir });
 });
 
 app.set('port', process.env.PORT || 8080);
 
-var server = app.listen(app.get('port'), function () {
-  console.log('listening on port ', server.address().port);
+var server = app.listen(app.get('port'), function (error) {
+  if (error) {
+    throw error;
+  }
```

Comment:
The new version of express now reports the errors to the callback, previously it was throwing directly.